extension/manifest.json

```diff
@@ -2,27 +2,40 @@
     "manifest_version": 2,
     "name": "BiReader",
     "version": "1.1",
-
     "description": "Based off of the principles of Bionic Reader to improve speed and depth of reading web text!",
-
     "icons": {
         "48": "icons/48.png",
         "96": "icons/96.png"
     },
-
     "browser_action": {
         "default_icon": "icons/32.png",
         "default_title": "Turn BiReader On"
     },
-
     "background": {
-        "scripts": ["browser-polyfill.js", "background.js"]
+        "scripts": [
+            "background.js"
+        ]
     },
-
     "content_scripts": [
         {
-            "matches": ["<all_urls>"],
-            "js": ["reader.js"]
+            "matches": [
+                "<all_urls>"
+            ],
+            "js": [
+                "reader.js"
+            ]
```

Comment:
These lines also don't need to be reformatted as the array can fit in one line. Having new lines for a single array element decreases readability